### ISSUE TITLE
Matches not count (no hieroglyph server crashes involved)

### ISSUE THREAD

**Issue by SIXSEVENONE** [ValveSoftware/Dota2-Gameplay#14345]
### Description

Match happened around 11 pm CET
Server address: A:1:1361386510:24867
Accounts involved - 244480824 and 918334673

Dire won the match with draft:
Slark, Outworld Devourer, Slardar, Phoenix, Treant Protector
Radiant lost the match with draft:
Juggernaut, Pangolier, Omniknight, Undying, Grimstroke

I still have the "reconnect" and "leave match" buttons
MMR number didn't get updated
My account id is 244480824
Attaching the screenshot and logs

### Screenshots

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/65928200/2f69c65e-7d45-44c3-9fd6-dfc0916ac175)
[reconnect_logs.log](https://github.com/ValveSoftware/Dota2-Gameplay/files/13694236/reconnect_logs.log)

**Comment by Popuyi** [comment]
it is exactly how it looks when somebody crashing with hieroglyphs, happened to me twice today both times leave game doesnt go away and game never appears when u leave.

**Comment by SIXSEVENONE** [comment]
> it is exactly how it looks when somebody crashing with hieroglyphs, happened to me twice today both times leave game doesnt go away and game never appears when u leave.

But this time noone used the all chat at the end of the game, maybe it works with teamchat? Or it may mean that another way to crash the server is present

**Comment by pashamykhailov** [comment]
Have the same!

**Comment by alexandershmatkov** [comment]
Same problem. 
https://steamcommunity.com/id/sashashmatkov/
played a hour ago. Player on broodmother started typing hieroglyphs in all chat. When game finished, there was an error about problems with connection to the server. In main menu "reconnect" and "leave match" buttons were available. I left match and it was not counted

**Comment by Vipeoo** [comment]
Yeah bro, same. Its allready second won ranked game that didn't count. thanks valve)

I just made a post describing all the details of the game that was crushed so guys pls comment this post and my to make sure valve will pay attention to this problem.
